hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -281,7 +281,7 @@ void clearMasterStubCache(MasterService.Interface stub) {
     masterStub.compareAndSet(stub, null);
   }
 
-  Optional<ServerStatisticTracker> getStatisticsTracker() {
+  ServerStatisticTracker getStatisticsTracker() {
```

Comment:
Why this change? It could be null and it is a return value, not a parameter...

Reply:
This is just to simplify as we are putting this returned value directly as argument to some methods.
e.g. 
https://github.com/apache/hbase/pull/673/files#diff-6705e0c1d8285fddc3d939ad3352b1a6R310
https://github.com/apache/hbase/pull/673/files#diff-e12d4030edb8885f06f86ee77cfb5b0cR383

Reply:
I think we should align the interface... It is a bit confusing that we return Optional for some methods, and then some other methods which do not return an Optional but could still return null...